lib/fog/orchestration/openstack/requests/create_stack.rb

```diff
@@ -1,3 +1,5 @@
+require "fog/orchestration/util/recursive_hot_file_loader"
```

Comment:
You can add the package in https://github.com/fog/fog-openstack/blob/master/lib/fog/openstack.rb then you don't need to `require` here and nor need to use the full namespace while using the resolver class.

Reply:
Rock! Thank you!!!